### ISSUE TITLE
Parse SDK 1.13.1 still uses OkHttp 2

### ISSUE THREAD

**Issue by igokoro** [ParsePlatform/Parse-SDK-Android#491]
Release notes for Parse SDK v. 1.13.1 state that Parse SDK now uses OkHttp3. But this is not what I see in the 1.13.1 jar file. After decompilation I see that `ParseHttpClient` still uses `com.squareup.okhttp` package to discover OkHttp - while OkHttp3 uses `okhttp3` package. Looks like something went wrong with the 1.13.1 build and publishing.

<img width="1440" alt="screen shot 2016-06-30 at 10 15 01 am" src="https://cloud.githubusercontent.com/assets/2870939/16491346/94b506e2-3eab-11e6-9d93-1e0e943afb0a.png">

**Comment by rogerhu** [comment]
There is a problem with the jCenter release: https://bintray.com/bintray/jcenter/com.parse%3Aparse-android/1.13.1#files/com/parse/parse-android/1.13.1

The checksum does not match.  The file hosted on Maven Central seems fine.

**Comment by rogerhu** [comment]
@grantland

**Comment by rogerhu** [comment]
The workaround is to specify mavenCentral() over jCenter() as the preferred repo in your allprojects:

```gradle
allprojects {
    repositories {
        mavenCentral() // switch back to jCenter() after Parse fixes
    }
}

You need to delete your Gradle cache possibly.  I nuked the `com.parse` directories and reloaded.

**Comment by igokoro** [comment]
@rogerhu This also happens with the JAR file I downloaded from Parse.com.

**Comment by rogerhu** [comment]
ping on this one again?

**Comment by grantland** [comment]
I've confirmed that the build on JCenter is incorrect and that using Maven Central is the correct workaround. I've sent a message to JCenter to resolve this, but it's out of my hands as we only directly upload to Maven Central. If this is still occurring after a few days, I'd recommend to reach out to JCenter yourselves.

@igokoro We're no longer updating parse.com with the latest artifacts and it seems you've downloaded 1.13.0 as 1.13.1 is not there. Please refer to Maven Central or this repository for future updates.

**Comment by rogerhu** [comment]
jCenter() resolved:

We appreciate your patience, we have resolved this issue. We believe the only way this could have occurred was that the offending jar was outdated in Maven Central for a brief period of time, during this time, Bintray cached the jar file. The cache has been cleared and this issue should not reoccur.

**Comment by danielapsmaior** [comment]
Hi,
I noticed that my code was referencing OkHttp2 today and tried many things to refresh the cache. 
Then I removed `maven { url 'https://maven.parse.com/repo' }` as it's not mentioned as necessary anymore and the repository can't be found..

What I have now is
```
repositories {
    mavenCentral()
    maven { url 'https://maven.fabric.io/public' }
    jcenter()
}
```
and `compile 'com.parse:parse-android:1.13.1'`
Tried `gradle --refresh-dependencies ` with no success.
The only way working now is by using the jar lib.
Could someone help me?
Thanks
